pkg/reboot/calculator.go

```diff
@@ -19,8 +19,8 @@ import (
 
 const (
 	MaxTimeForNoPeersResponse = 30 * time.Second
-	NodesNumberInFirstBatch = 3
-	MaxBatchesAfterFirst    = 10
+	MinNodesNumberInBatch     = 3
+	MaxBatchesAfterFirst      = 10
```

Comment:
`MaxBatchesAfterFirst`? After first batch? I don't understand the variable name. 
Can it just be `MaxBatches`?

Reply:
First batch is always of 3 nodes. this parameter refers to the number of batches that follow the first one.
For example for 53 nodes we'd have 1 batch of 3 nodes and 10 more batches of 5 nodes each, so a total of 53 nodes in 11 batches .